state-recover/besu-plugin/src/main/kotlin/linea/staterecover/plugin/RecoveryModeManager.kt

```diff
@@ -81,8 +73,8 @@ class RecoveryModeManager(
 
   /** Switches the node to recovery mode.  */
   private fun switchToRecoveryMode() {
-    check(recoveryModeState == RecoveryModeState.NORMAL_MODE) {
-      "Cannot switch to recovery mode from state: $recoveryModeState"
+    check(recoveryModeTriggered.get()) {
```

Comment:
Shouldn't this be `check(!recoveryModeTriggered.get()`)?

Reply:
Yes it should. Thanks